doajtest/fixtures/applications.py

```diff
@@ -35,21 +38,28 @@ def make_many_application_sources(count=2, in_doaj=False):
         return application_sources
 
     @staticmethod
-    def make_application_form():
-        return deepcopy(APPLICATION_FORM)
+    def make_application_form(role="maned"):
+        form = deepcopy(APPLICATION_FORM)
+        if role == "assed" or role == "editor":
```

Comment:
looks like in the fixtures we've chosen overall to go with the short forms ("maned", "assed"), which does make them different from the rest of the system.  Something to tidy up when we review the fixtures, rather than this PR, I think.